### ISSUE TITLE
Example streaming text to speech code ain't working

### ISSUE THREAD

**Issue by didemkkaslan** [elevenlabs/elevenlabs-js#78]
I'm trying to use the code inside `text_to_speech_stream.ts` to get audio stream and upload it on aws s3 but it gives me below error. 
```

/**
 * Uses the ElevenLabs API to convert text to audio
 * @param text  The text to convert to audio
 * @returns The URL of the audio file
 */
export const textToSpeech: (text: string) => Promise<Buffer> = async (
  text: string
) => {
  return new Promise(async (resolve, reject) => {
    try {
      const audioStream = await elabs.generate({
        voice: "Rachel",
        model_id: "eleven_turbo_v2",
        text,
      });

      console.log("audioStream", audioStream);
      console.log(typeof audioStream);

      const chunks: Buffer[] = [];

      for await (const chunk of audioStream) {
        chunks.push(chunk);
      }

      const audio = Buffer.concat(chunks);

      resolve(audio);
    } catch (error) {
      console.log("error", error);
      reject(error);
    }
  });
};
```

<img width="963" alt="Screenshot 2024-08-05 at 13 44 05" src="https://github.com/user-attachments/assets/d885e0bd-5940-41e5-82ac-1121b553c173">

**Comment by dsinghvi** [comment]
@didemkkaslan this should be fixed in the latest 0.14.0 release (just published). Please let us know if you still have issues!

**Comment by didemkkaslan** [comment]
Thanks @dsinghvi  I've updated to the latest release 0.14.0 and problem is solved for me 🎊
